infrastructure/azure.tf

```diff
@@ -79,6 +79,12 @@ variable "azurerm_storage_queue_profileevents" {
   description = "Name of the storage queue for profile events (create / update)"
 }
 
+variable "azurerm_storage_container_cached" {
+  type        = "string"
+  default     = "cached"
```

Comment:
I wonder if `cached` is a little bit too generic, what about `cached-data` or `functions-cache`?

Reply:
I like `cached-data` but we have to:
1) Change the const here https://github.com/teamdigitale/io-functions-commons/blob/master/src/models/visible_service.ts#L15
2) Move/copy the current visible-services.json from `cached` to `cached-data`

P.S. I also like something with `projections` in the Event Sourcing meaning.

Reply:
This will hopefully last just few more days / weeks. I'd propose to keep going, merge and port it on the new infrastructure asap.

Reply:
yeah let's keep the changes at minimum, we can fix it later